services/QuillLMS/app/views/pages/shared/_assign_activity.html.erb

```diff
@@ -0,0 +1,21 @@
+<% unless local_assigns.key?(:tool) %>
+  <% raise "Missing required local variable: tool" %>
+<% end %>
+
+<% assign_url = "#{ENV['DEFAULT_URL']}/assign/activity-library?activityClassificationFilters[]=#{tool}" %>
+
+<% if current_user %>
+  <section class="tool-sign-up">
+    <h1 class="q-h1">Get started now! Assign Quill <%= tool.capitalize %>.</h1>
+    <div>
+      <%= link_to "Assign", assign_url, class: "quill-button contained large white focus-on-dark" %>
+    </div>
+  </section>
+<% else %>
+  <section class="tool-sign-up">
+    <h1 class="q-h1">Sign up to start using Quill <%= tool.capitalize %> in your class today</h1>
```

Comment:
I think we end up with slightly weird behavior here, like when the tool's key doesn't match the tool's name (ie, `passage` for `Proofreader`).